btc-withdrawal/src/main/kotlin/com/d3/btc/withdrawal/handler/NewTransferHandler.kt

```diff
@@ -16,9 +20,12 @@ import java.math.BigDecimal
  */
 @Component
 class NewTransferHandler(
```

Comment:
So this class only checks if a transfer is the withdrawal trigger, shouldn't it be renamed accordingly?